.env

```diff
@@ -96,8 +96,8 @@ DEVTOOLSET_VERSION=
 # Please also update the crossbow configuration in order to keep the github
 # actions cache up to date for the macOS wheels:
 #   https://github.com/ursacomputing/crossbow/blob/master/.github/workflows/cache_vcpkg.yml
```

Comment:
Oh, it seems that this is outdated.
https://github.com/ursacomputing/crossbow/pull/7 automated this.

Reply:
I removed this comment in https://github.com/apache/arrow/pull/35430. I suggest continuing the discussion there?